spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpNioClientConnectionFactory.java

```diff
@@ -113,6 +115,11 @@ protected TcpConnectionSupport buildNewConnection() {
 			return wrappedConnection;
 		}
 		catch (IOException e) {
+			try {
+				socketChannel.close();
+			} catch (IOException e2) {
```

Comment:
Our code style is to have `catch` on a new line.
You can run locally `./gradlew :spring-integration-ip:check`.

Reply:
Checked the style guide and fixed it accordingly. Thanks for the feedback :)